Makefile

```diff
@@ -0,0 +1,94 @@
+.PHONY: clean clean-test clean-pyc clean-build docs help
```

Comment:
This would be useful, but on a different PR (I'd be fine to see this with the changes to `.dockerignore`)